README.md

```diff
@@ -222,3 +221,21 @@ Example
 > net config-interface ifname=eth1 dhcpv4enabled=false ipv4addr=192.168.1.1 ipv4mask=24 clearaddresses=true
 [OK] 200 OK
 ```
+
+
+# Developing Maestro-Shell
+
+### Compiling
+1. Since Maestro-Shell is the front end for Maestro you need to first follow all the current build instruction to build [Maestro](https://github.com/armPelionEdge/maestro/tree/dev).
```

Comment:
Let's just link to the Maestro repo (https://github.com/armPelionEdge/maestro) and leave out the branch name. If we left the branch name in place here, we would need to remember to change it when we merge to master at the end of next sprint.

Reply:
will do.

Reply:
done.